src/rules/utils.ts

```diff
@@ -550,7 +550,15 @@ export enum TestCaseProperty {
   'todo' = 'todo',
 }
 
-export type JestFunctionName = DescribeAlias | TestCaseName | HookName;
+export enum TopCase {
+  'top' = 'top',
+}
+
+export type JestFunctionName =
+  | DescribeAlias
+  | TestCaseName
+  | HookName
+  | TopCase;
```

Comment:
`top` is not a valid function provided by `Jest`, so it shouldn't be in this union.